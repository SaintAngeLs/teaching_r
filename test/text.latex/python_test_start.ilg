This is makeindex, version 2.16 [TeX Live 2022] (kpathsea + Thai support).
Scanning input file python_test_start.idx...done (0 entries accepted, 0 rejected).
Nothing written in python_test_start.ind.
Transcript written in python_test_start.ilg.
