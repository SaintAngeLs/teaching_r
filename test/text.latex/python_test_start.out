\BOOKMARK [1][-]{section.1}{\376\377\000T\000a\000s\000k\000s\000\040\000S}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000T\000a\000s\000k\000\040\000S\0001}{section.1}% 2
\BOOKMARK [3][-]{subsubsection.1.1.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.1}% 3
\BOOKMARK [3][-]{subsubsection.1.1.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.1}% 4
\BOOKMARK [3][-]{subsubsection.1.1.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.1}% 5
\BOOKMARK [3][-]{subsubsection.1.1.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.1}% 6
\BOOKMARK [3][-]{subsubsection.1.1.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.1}% 7
\BOOKMARK [2][-]{subsection.1.2}{\376\377\000T\000a\000s\000k\000\040\000S\0002}{section.1}% 8
\BOOKMARK [3][-]{subsubsection.1.2.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.2}% 9
\BOOKMARK [3][-]{subsubsection.1.2.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.2}% 10
\BOOKMARK [3][-]{subsubsection.1.2.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.2}% 11
\BOOKMARK [3][-]{subsubsection.1.2.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.2}% 12
\BOOKMARK [3][-]{subsubsection.1.2.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.2}% 13
\BOOKMARK [2][-]{subsection.1.3}{\376\377\000T\000a\000s\000k\000\040\000S\0003}{section.1}% 14
\BOOKMARK [3][-]{subsubsection.1.3.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.3}% 15
\BOOKMARK [3][-]{subsubsection.1.3.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.3}% 16
\BOOKMARK [3][-]{subsubsection.1.3.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.3}% 17
\BOOKMARK [3][-]{subsubsection.1.3.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.3}% 18
\BOOKMARK [3][-]{subsubsection.1.3.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.3}% 19
\BOOKMARK [2][-]{subsection.1.4}{\376\377\000T\000a\000s\000k\000\040\000S\0004}{section.1}% 20
\BOOKMARK [3][-]{subsubsection.1.4.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.4}% 21
\BOOKMARK [3][-]{subsubsection.1.4.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.4}% 22
\BOOKMARK [3][-]{subsubsection.1.4.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.4}% 23
\BOOKMARK [3][-]{subsubsection.1.4.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.4}% 24
\BOOKMARK [3][-]{subsubsection.1.4.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.4}% 25
\BOOKMARK [2][-]{subsection.1.5}{\376\377\000T\000a\000s\000k\000\040\000S\0005}{section.1}% 26
\BOOKMARK [3][-]{subsubsection.1.5.1}{\376\377\000T\000a\000s\000k\000\040\0001\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.5}% 27
\BOOKMARK [3][-]{subsubsection.1.5.2}{\376\377\000T\000a\000s\000k\000\040\0002\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.5}% 28
\BOOKMARK [3][-]{subsubsection.1.5.3}{\376\377\000T\000a\000s\000k\000\040\0003\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.5}% 29
\BOOKMARK [3][-]{subsubsection.1.5.4}{\376\377\000T\000a\000s\000k\000\040\0004\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.5}% 30
\BOOKMARK [3][-]{subsubsection.1.5.5}{\376\377\000T\000a\000s\000k\000\040\0005\000\040\000\050\0001\0000\000\040\000p\000o\000i\000n\000t\000s\000\051}{subsection.1.5}% 31
\BOOKMARK [1][-]{section.2}{\376\377\000T\000a\000s\000k\000s\000\040\000P}{}% 32
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0001\000:\000\040\000T\000u\000r\000t\000l\000e\000\040\000d\000r\000a\000w\000i\000n\000g\000\040}{section.2}% 33
\BOOKMARK [3][-]{subsubsection.2.1.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.2.1}% 34
\BOOKMARK [3][-]{subsubsection.2.1.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n}{subsection.2.1}% 35
\BOOKMARK [2][-]{subsection.2.2}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0002\000:\000\040\000N\000u\000m\000b\000e\000r\000\040\000G\000u\000e\000s\000s\000i\000n\000g\000\040\000G\000a\000m\000e}{section.2}% 36
\BOOKMARK [3][-]{subsubsection.2.2.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.2.2}% 37
\BOOKMARK [3][-]{subsubsection.2.2.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000s}{subsection.2.2}% 38
\BOOKMARK [2][-]{subsection.2.3}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0003\000:\000\040\000T\000o\000-\000D\000o\000\040\000L\000i\000s\000t}{section.2}% 39
\BOOKMARK [3][-]{subsubsection.2.3.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.2.3}% 40
\BOOKMARK [3][-]{subsubsection.2.3.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000s}{subsection.2.3}% 41
\BOOKMARK [2][-]{subsection.2.4}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0004\000:\000\040\000S\000i\000m\000p\000l\000e\000\040\000C\000a\000l\000c\000u\000l\000a\000t\000o\000r}{section.2}% 42
\BOOKMARK [3][-]{subsubsection.2.4.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.2.4}% 43
\BOOKMARK [3][-]{subsubsection.2.4.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000s}{subsection.2.4}% 44
\BOOKMARK [2][-]{subsection.2.5}{\376\377\000P\000r\000o\000j\000e\000c\000t\000\040\0005\000:\000\040\000H\000a\000n\000g\000m\000a\000n\000\040\000G\000a\000m\000e}{section.2}% 45
\BOOKMARK [3][-]{subsubsection.2.5.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{subsection.2.5}% 46
\BOOKMARK [3][-]{subsubsection.2.5.2}{\376\377\000S\000p\000e\000c\000i\000f\000i\000c\000a\000t\000i\000o\000n\000s}{subsection.2.5}% 47
\BOOKMARK [1][-]{section.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S}{}% 48
\BOOKMARK [2][-]{subsection.3.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S\0001}{section.3}% 49
\BOOKMARK [3][-]{subsubsection.3.1.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0001\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.1}% 50
\BOOKMARK [3][-]{subsubsection.3.1.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0002\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.1}% 51
\BOOKMARK [3][-]{subsubsection.3.1.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0003\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.1}% 52
\BOOKMARK [3][-]{subsubsection.3.1.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0004\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.1}% 53
\BOOKMARK [3][-]{subsubsection.3.1.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0005\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.1}% 54
\BOOKMARK [2][-]{subsection.3.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S\0002}{section.3}% 55
\BOOKMARK [3][-]{subsubsection.3.2.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0001\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.2}% 56
\BOOKMARK [3][-]{subsubsection.3.2.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0002\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.2}% 57
\BOOKMARK [3][-]{subsubsection.3.2.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0003\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.2}% 58
\BOOKMARK [3][-]{subsubsection.3.2.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0004\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.2}% 59
\BOOKMARK [3][-]{subsubsection.3.2.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0005\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.2}% 60
\BOOKMARK [2][-]{subsection.3.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S\0003}{section.3}% 61
\BOOKMARK [3][-]{subsubsection.3.3.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0001\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.3}% 62
\BOOKMARK [3][-]{subsubsection.3.3.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0002\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.3}% 63
\BOOKMARK [3][-]{subsubsection.3.3.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0003\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.3}% 64
\BOOKMARK [3][-]{subsubsection.3.3.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0004\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.3}% 65
\BOOKMARK [3][-]{subsubsection.3.3.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0005\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.3}% 66
\BOOKMARK [2][-]{subsection.3.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S\0004}{section.3}% 67
\BOOKMARK [3][-]{subsubsection.3.4.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0001\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.4}% 68
\BOOKMARK [3][-]{subsubsection.3.4.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0002\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.4}% 69
\BOOKMARK [3][-]{subsubsection.3.4.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0003\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.4}% 70
\BOOKMARK [3][-]{subsubsection.3.4.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0004\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.4}% 71
\BOOKMARK [3][-]{subsubsection.3.4.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0005\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.4}% 72
\BOOKMARK [2][-]{subsection.3.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000S\0005}{section.3}% 73
\BOOKMARK [3][-]{subsubsection.3.5.1}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0001\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.5}% 74
\BOOKMARK [3][-]{subsubsection.3.5.2}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0002\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.5}% 75
\BOOKMARK [3][-]{subsubsection.3.5.3}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0003\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.5}% 76
\BOOKMARK [3][-]{subsubsection.3.5.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0004\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.5}% 77
\BOOKMARK [3][-]{subsubsection.3.5.5}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\0005\000\040\000\050\0001\0000\000\040\004\061\004\060\004\073\004\126\004\062\000\051}{subsection.3.5}% 78
\BOOKMARK [1][-]{section.4}{\376\377\004\027\004\060\004\062\004\064\004\060\004\075\004\075\004\117\000\040\000P}{}% 79
\BOOKMARK [2][-]{subsection.4.1}{\376\377\004\037\004\100\004\076\004\065\004\072\004\102\000\040\0001\000:\000\040\004\034\004\060\004\073\004\116\004\062\004\060\004\075\004\075\004\117\000\040\004\067\004\060\000\040\004\064\004\076\004\077\004\076\004\074\004\076\004\063\004\076\004\116\000\040\000T\000u\000r\000t\000l\000e}{section.4}% 80
\BOOKMARK [3][-]{subsubsection.4.1.1}{\376\377\004\036\004\077\004\070\004\101}{subsection.4.1}% 81
\BOOKMARK [3][-]{subsubsection.4.1.2}{\376\377\004\041\004\077\004\065\004\106\004\070\004\104\004\126\004\072\004\060\004\106\004\126\004\117}{subsection.4.1}% 82
\BOOKMARK [2][-]{subsection.4.2}{\376\377\004\037\004\100\004\076\004\065\004\072\004\102\000\040\0002\000:\000\040\004\023\004\100\004\060\000\040\004\062\004\063\004\060\004\064\004\103\004\062\004\060\004\075\004\075\004\117\000\040\004\107\004\070\004\101\004\065\004\073}{section.4}% 83
\BOOKMARK [3][-]{subsubsection.4.2.1}{\376\377\004\036\004\077\004\070\004\101}{subsection.4.2}% 84
\BOOKMARK [3][-]{subsubsection.4.2.2}{\376\377\004\041\004\077\004\065\004\106\004\070\004\104\004\126\004\072\004\060\004\106\004\126\004\117}{subsection.4.2}% 85
\BOOKMARK [2][-]{subsection.4.3}{\376\377\004\037\004\100\004\076\004\065\004\072\004\102\000\040\0003\000:\000\040\004\041\004\077\004\070\004\101\004\076\004\072\000\040\004\101\004\077\004\100\004\060\004\062}{section.4}% 86
\BOOKMARK [3][-]{subsubsection.4.3.1}{\376\377\004\036\004\077\004\070\004\101}{subsection.4.3}% 87
\BOOKMARK [3][-]{subsubsection.4.3.2}{\376\377\004\041\004\077\004\065\004\106\004\070\004\104\004\126\004\072\004\060\004\106\004\126\004\117}{subsection.4.3}% 88
\BOOKMARK [2][-]{subsection.4.4}{\376\377\004\037\004\100\004\076\004\065\004\072\004\102\000\040\0004\000:\000\040\004\037\004\100\004\076\004\101\004\102\004\070\004\071\000\040\004\072\004\060\004\073\004\114\004\072\004\103\004\073\004\117\004\102\004\076\004\100}{section.4}% 89
\BOOKMARK [3][-]{subsubsection.4.4.1}{\376\377\004\036\004\077\004\070\004\101}{subsection.4.4}% 90
\BOOKMARK [3][-]{subsubsection.4.4.2}{\376\377\004\041\004\077\004\065\004\106\004\070\004\104\004\126\004\072\004\060\004\106\004\126\004\117}{subsection.4.4}% 91
\BOOKMARK [2][-]{subsection.4.5}{\376\377\004\037\004\100\004\076\004\065\004\072\004\102\000\040\0005\000:\000\040\004\023\004\100\004\060\000\040\004\022\004\126\004\101\004\126\004\073\004\073\004\070\004\106\004\117}{section.4}% 92
\BOOKMARK [3][-]{subsubsection.4.5.1}{\376\377\004\036\004\077\004\070\004\101}{subsection.4.5}% 93
\BOOKMARK [3][-]{subsubsection.4.5.2}{\376\377\004\041\004\077\004\065\004\106\004\070\004\104\004\126\004\072\004\060\004\106\004\126\004\127}{subsection.4.5}% 94
\BOOKMARK [2][-]{subsection.4.6}{\376\377\000A\000d\000d\000i\000t\000i\000o\000n\000a\000l\000\040\000r\000e\000f\000e\000r\000e\000n\000c\000e\000s}{section.4}% 95
\BOOKMARK [2][-]{subsection.4.7}{\376\377\000A\000\040\000D\000e\000e\000p\000\040\000D\000i\000v\000e\000\040\000i\000n\000t\000o\000\040\000t\000h\000e\000\040\000Q\000u\000a\000d\000r\000a\000t\000i\000c\000\040\000E\000q\000u\000a\000t\000i\000o\000n}{section.4}% 96
\BOOKMARK [2][-]{subsection.4.8}{\376\377\000S\000o\000l\000v\000i\000n\000g\000\040\000S\000e\000c\000o\000n\000d\000\040\000O\000r\000d\000e\000r\000\040\000E\000q\000u\000a\000t\000i\000o\000n\000s}{section.4}% 97
